src/index.html

```diff
@@ -7,6 +7,36 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day start-day_sun"></div>
```

Comment:
`start-day_sun` should be a modifier.

More about naming you could watch  [there](https://mate.academy/learn/css-advanced/bem#/video/920)
timing 0:30

```suggestion
    <div class="calendar">
      <div class="calendar__day calendar__day--start-day_sun"></div>
```

Reply:
> Add a modifier month-length for the calendar with values 28, 29, 30 and 31 (use @for) It sets the last day to show (use [nth-child](https://css-tricks.com/how-nth-child-works/))

Ok, I don't sure that i understand this task correctly, but I solve as I can.

Reply:
Also it's strange, because I don't have any problems with tests before I  take a pull request